port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -230,6 +251,10 @@ async def _parse_items(
                         examples_to_send.append(result.raw_data)
                 else:
                     failed_entities.append(parsed_entity)
+        if len(entity_misconfigurations) > 0:
+            logger.info(
```

Comment:
I think that showing it here is too verbose, let's move it to be per batch of entities instead of every entity